src/main/java/org/example/Exceptions/Entity.java

```diff
@@ -0,0 +1,15 @@
+package org.example.Exceptions;
+
+public enum Entity {
```

Comment:
Entity? The name is throwing me off

Reply:
re named to Error entity as it refers to the entity the error occurred with